content/blog/blog-post-08.md

```diff
@@ -36,8 +37,7 @@ The Advisory Council is also the primary body to organize day-to-day collaborati
 
 Key benefits of belonging to the High-Performance Software Foundation include open governance models, resources for improving software quality and building community, as well as collaborations with the larger Linux computing ecosystem.
 Leveraging well established open governance models from the Linux Foundation, HPSF provides a framework to organize multi-institutional collaborations.
-An institution that wants to leverage Kokkos and also become contributors can now rest assured that the project is not just subject to the whims of some project leads paid for by DOE.
-Donating rights to Kokkos to the Linux Foundation also demonstrates that the Kokkos team and DOE has been serious with their intent to evolve Kokkos into a true community project.
+Donating rights to Kokkos to the Linux Foundation also demonstrates that the Kokkos team and DOE are serious with their intent to evolve Kokkos into a true community project.
```

Comment:
Rather than bringing intention into the discussion, would it make more sense to talk about reducing uncertainty / risk around adopting Kokkos, because the Linux Foundation, who now own the project , are in the business of providing long-term stability / maintenance / availability?  

Maybe something along the lines of :

"Joining the Linux Foundation ensures Kokkos' long-term health, quality and availability for the R&D community"